ds-controller-board/res/configuration.toml

```
@@ -85,4 +78,12 @@ PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name
   Retained = "false"
   AutoReconnect = "true"
   ConnectTimeout = "5" # Seconds
-  SkipCertVerify = "false"
\ No newline at end of file
+  SkipCertVerify = "false"
+
+[AppCustom]
+  [DriverConfig]
+  VID = '2341'
+  PID = '8037'
+  DisplayTimeout = '10s'
+  LockTimeout = '30s'
+  VirtualControllerBoard = 'true'
```

The reason to make this custom config is we can have typed data rather than all strings.
```suggestion
  VirtualControllerBoard = true
```